pages/KeyConceptsInOppia.html

```diff
@@ -69,7 +69,7 @@ <h2>
     of the above conversation:
   </p>
 
-  <img src="static/images/threeBalls.png" width="75%;">
+  <img src="static/images/threeBalls.png" width="75%;" alt="Three balls (red, blue and green) overlapping each other. Below which is the problem statement followed by a button titled 'Not sure where to start?', a text-area for the learner to enter his/her answer and a white submit button. Another view of the same card is immediately below, prompting the learner to enter all of their answers, having the 3 balls arranged in a linear way of the order red - blue - green. Also present is a text-area for the learner to enter his answers and a white 'submit' button.">
```

Comment:
I think if you say this out loud to someone else and ask them to draw it (in their mind or on paper), they will not be able to. Could you try to summarize the problem statement?

Also, we typically go with singular 'they' rather than his/her. And "their" rather than "his" (in the last sentence).

Note that the colours of the buttons probably don't matter, given the context.